modules/portal/portal-impl/src/main/java/com/enonic/xp/portal/impl/handler/SlashApiHandler.java

```diff
@@ -252,15 +252,18 @@ private boolean verifyPathMountedOnWebapps( final DescriptorKey descriptorKey, f
 
     private boolean verifyRequestMountedOnSites( final DescriptorKey descriptorKey, final PortalRequest portalRequest )
     {
+        final ContentResolverResult contentResolverResult = new ContentResolver( contentService ).resolve( portalRequest );
```

Comment:
I don't think media handlers ever had access to site/content

Reply:
This is done in the site context block, which will not be applicable to the slash API part.